docs/admin/_kubeadm/kubeadm.md

```diff
@@ -0,0 +1,60 @@
+## kubeadm
+
+kubeadm: easily bootstrap a secure Kubernetes cluster
+
+### Synopsis
+
+
+
+kubeadm: easily bootstrap a secure Kubernetes cluster.
+
+    ┌──────────────────────────────────────────────────────────┐
+    │ KUBEADM IS BETA, DO NOT USE IT FOR PRODUCTION CLUSTERS!  │
```

Comment:
Can we change this from "DO NOT USE" to something a bit softer, like "kubeadm is beta, use at your own risk" or "kubeadm is currently in beta"?

Reply:
@roberthbailey this is part of the generate docs. I will take charge to implement this suggesting on the corresponding code in kubernetes repo

Reply:
really good point @roberthbailey, we REALLY should reword this, had totally forgot about it

Reply:
+1 to re-wording "production" is a ill-defined term btw.